packages/admin-rte/src/core/Controls/BlockTypesControls.tsx

```diff
@@ -39,7 +34,39 @@ export default function BlockTypesControls({
                         {c.label}
                     </MenuItem>
                 ))}
-            </sc.Select>
+            </Select>
         </FormControl>
     );
 }
+
+export type CometAdminRteBlockTypeControlsClassKeys = "root" | "select";
+
+const styles = (theme: Theme) =>
+    createStyles<CometAdminRteBlockTypeControlsClassKeys, any>({
+        root: {},
+        select: {
+            color: theme.rte.colors.buttonIcon,
+            minWidth: 180,
+            lineHeight: "24px",
+            fontSize: 14,
+            padding: 0,
+            "& [class*='MuiSvgIcon-root']": {
+                color: "inherit",
+            },
+        },
+    });
+
+const StyledBlockTypesControls = withStyles(styles, { name: "CometAdminRteBlockTypeControls" })(BlockTypesControls);
```

Comment:
Is the name.option needed/useful? The docs say this option is for debugging: https://material-ui.com/styles/api/#arguments-5

Reply:
Yes, it defines the key for theme overrides/props.